client/index.html

```diff
@@ -5,6 +5,9 @@
   <meta name="color-scheme" content="dark light">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Browserslist</title>
+  <link rel="preload" href="./view/base/font-lato/Lato-Regular.woff2" as="font" type="font/woff2" crossorigin="crossorigin" />
+  <link rel="preload" href="./view/base/font-lato/Lato-Bold.woff2" as="font" type="font/woff2" crossorigin="crossorigin" />
+  <link rel="preload" href="./view/base/font-lato/Lato-Black.woff2" as="font" type="font/woff2" crossorigin="crossorigin" />
```

Comment:
Can we reduce `Black` variant? It will reduce size by 1/3.

Reply:
![2022-08-16_14-51-43](https://user-images.githubusercontent.com/22644149/184883428-810efdb1-a0e7-4c99-9abd-71b92ecf0c6c.png)